src/Neo/SmartContract/StorageItem.cs

```diff
@@ -193,5 +193,15 @@ public void Set(IInteroperable interoperable)
             item.cache ??= new BigInteger(item.value.Span);
             return (BigInteger)item.cache;
         }
+
+        public static implicit operator StorageItem(BigInteger value)
```

Comment:
I'm just wondering, do you have a use-case for these conversion operators in the code? Usually when we introduce such helpers, there's a need to use them in the code and we replace some real code with these helpers.

Reply:
Doesn't matter how it done. As long as the do the same thing. We have direct access to the `class`. So, no need for extension methods.

Reply:
> I'm just wondering, do you have a use-case for these conversion operators in the code? Usually when we introduce such helpers, there's a need to use them in the code and we replace some real code with these helpers.

Does it matter to you?